test/ProductConstructionService.ScenarioTests/TestParameters.cs

```diff
@@ -138,6 +142,31 @@ private static async Task InstallDarc(IProductConstructionServiceApi pcsApi, Sha
         await TestHelpers.RunExecutableAsync(dotnetExe, [.. toolInstallArgs]);
     }
 
+    private static async Task<string?> TryGetGitHubTokenFromCliAsync()
+    {
+        var logger = NullLoggerFactory.Instance.CreateLogger(nameof(ProcessManager));
```

Comment:
Do the agents running scenario tests even have this installed? I thought we'd continue to use the token that only has permissions in maestro-test-auth, if so we don't really need this method?

Reply:
Your local does